SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,93 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition []'),'2;1');
+FILE_NAME('SectionedSurface-Aplitop-1.ifc','2021-04-06T09:56:30',(),(),'IfcOpenShell 0.5.0-dev','IfcOpenShell 0.5.0-dev','');
+FILE_SCHEMA(('IFC4X3_RC3'));
+ENDSEC;
+DATA;
+#1=IFCACTORROLE(.USERDEFINED.,'',$);
+#2=IFCPOSTALADDRESS(.HOME.,$,$,$,(''),$,$,$,$,$);
+#3=IFCPERSON('Javier','','',$,$,$,(#1),(#2));
+#4=IFCPOSTALADDRESS(.OFFICE.,$,$,$,(''),$,$,$,$,$);
+#5=IFCORGANIZATION('','Aplitop','',$,(#4));
+#6=IFCPOSTALADDRESS(.OFFICE.,$,$,'',(''),$,'','','','');
+#7=IFCAPPLICATION(#5,'8.5','TCP-Digital Terrain Model','TcpMDT');
+#8=IFCPERSONANDORGANIZATION(#3,#5,$);
+#9=IFCOWNERHISTORY(#8,#7,$,$,$,$,$,0);
+#10=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#11=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#12=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#13=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#14=IFCUNITASSIGNMENT((#10,#11,#12,#13));
+#15=IFCDIRECTION((1.,0.,0.));
+#16=IFCDIRECTION((0.,0.,1.));
+#17=IFCCARTESIANPOINT((0.,0.,0.));
+#18=IFCAXIS2PLACEMENT3D(#17,#16,#15);
+#19=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#18,$);
+#20=IFCDIRECTION((1.,0.,0.));
+#21=IFCDIRECTION((0.,0.,1.));
+#22=IFCCARTESIANPOINT((0.,0.,0.));
+#23=IFCAXIS2PLACEMENT3D(#22,#21,#20);
+#24=IFCLOCALPLACEMENT($,#23);
+#25=IFCSITE('2zHKom8bz2oQNeL25oFkgh',#9,'Site',$,$,#24,$,$,.ELEMENT.,$,$,$,$,#2);
+#26=IFCPROJECT('0VGYxaGHb3APEG3MpCN2LN',#9,'Project','',$,'',$,$,#14);
+#27=IFCRELAGGREGATES('3s8mCI$QTDxwH4srCKLj04',#9,$,$,#26,(#25));
+#28=IFCROAD('2VBIkfjsriouJ$gISkmggg',#9,'Road','','ObjectType',#24,#272,'LongName',.COMPLEX.,.USERDEFINED.);
+#29=IFCRELAGGREGATES('3Z1$Qqe6XEcQRCqJoCdqUM',#9,$,$,#25,(#28));
+#30=IFCALIGNMENT('1hmNMQuc53NAma6AFvRPA6',$,'Eje1 (1)',$,$,#24,#49,$);
+#31=IFCALIGNMENTHORIZONTAL('2LpapCuJP0I8eaolX4R5iL',$,'Eje1 (1)',$,$,#24,$,0.);
+#32=IFCRELNESTS('2q4l6QB$b5o8KpfezYsjnf',#9,$,$,#30,(#31,#51));
+#33=IFCALIGNMENTSEGMENT('36ZpVDkF51Xh2KpEiWEhlp',$,$,$,$,#24,$,#35);
+#34=IFCCARTESIANPOINT((286.47559897,239.79766567));
+#35=IFCALIGNMENTHORIZONTALSEGMENT($,$,#34,1.50346536986339,0.,0.,107.15812752,$,.LINE.);
+#36=IFCDIRECTION((0.999655739909883,0.0262374096515884));
+#37=IFCAXIS2PLACEMENT2D(#34,#36);
+#40=IFCCARTESIANPOINT((0.,0.));
+#41=IFCDIRECTION((1.,0.));
+#42=IFCVECTOR(#41,1.);
+#43=IFCLINE(#40,#42);
+#44=IFCCURVESEGMENT(.CONTINUOUS.,#37,IFCNONNEGATIVELENGTHMEASURE(0.),IFCNONNEGATIVELENGTHMEASURE(107.15812752),#43);
+#45=IFCRELNESTS('1qVzEdro5CFf5K20Wsg4rv',#9,$,$,#31,(#33));
+#46=IFCCOMPOSITECURVE((#44),.F.);
+#47=IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#19,$,.GRAPH_VIEW.,$);
+#48=IFCSHAPEREPRESENTATION(#47,'Axis','Curve2D',(#46));
```

Comment:
When there is an IfcGradientCurve available, I wonder if the shape representation for an IfcAlignment should refer to this  (65) instead if the IfcCompositeCurve?

Reply:
In a simple sample like this, just a straight line, yes. 

But in a more general sample, I still don't have an implementation to build a single IfcCompositeCurve from any 2D alingment and grade line info. So I kept alignment representation as 2D.

I will change it.

Reply:
It seems as if the IfcProductDefinitionShape on line #49 no longer has an owning IfcProduct. I assume that this should be the IfcAlignment?

Reply:
I see that IfcAlignmentHorizontal has a shape representation = IfcCompositeCurve.
IfcAlignmentVertical has a shape representation = IfcGradientCurve
I wonder whether IfcAlignment (no shape representation in this file) should have the gradient curve as its representation (Axis,Curve3D)? @SergejMuhic ?